product_multi_lot_sequence/views/product_views.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <!-- view common to both template and product -->
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="name">product.template.stock.property.form.inherit</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view" />
+        <field name="arch" type="xml">
+            <page name="inventory" position="after">
+                <page
+                    name="tracking"
+                    string="Tracking"
+                    attrs="{'invisible': [('tracking', '=', 'none')]}"
+                >
+                    <group string="Product Lot Sequence" name="product_lot_sequence">
+                        <field
+                            name="product_lot_sequence_ids"
+                            context="{'default_product_tmpl_id': context.get('product_tmpl_id', active_id)}"
```

Comment:
I'm also having some issues when creating a product lot sequences from the one2many field, product_tmpl_id is not assigned correctly when opening the new record form.